test/com/ableton/VirtualEnvTest.groovy

```diff
@@ -50,13 +51,15 @@ class VirtualEnvTest extends BasePipelineTest {
   @Test
   void newObjectWithNullPython() throws Exception {
     try {
-      def venv = new VirtualEnv(script, null)
+      @SuppressWarnings('UnusedVariable')
+      def unused = new VirtualEnv(script, null)
       fail('Expected exception, but none was thrown')
     } catch (AssertionError error) {
       assertNotNull(error)
     }
   }
 
+  @SuppressWarnings('BuilderMethodWithSideEffects')
```

Comment:
Could this be moved closer to the line that triggers the warning?

Reply:
Yes, it can be moved 1 line down, I guess.